docs/2_2_common_mechanisms.adoc

```diff
@@ -1040,15 +1040,14 @@ where
 ** <<StepMode>> (Co-Simulation): The variables listed as elements <<Output>> of `<ModelStructure>` with a floating point type and <<variability>> = <<continuous>> or <<discrete>>.
 Each state derivative variable listed as elements <<StateDerivative>> in `<ModelStructure>`, if present.
 
-* latexmath:[\mathbf{v}_{\mathit{known}}] is the vector of known <<input>> variables of function *f* that changes its value in the actual state.
+* latexmath:[\mathbf{v}_{\mathit{known}}] is the vector of <<input>> variables of function *f* which can be changed by the importer in the current state.
 Details about which variables are in latexmath:[\mathbf{v}_{\mathit{known}}] are given in the description of element <<dependencies>> in <<ModelStructure>>.
 
-* latexmath:[{\mathbf{v}_{\mathit{rest}}}] is the set of <<input>> variables of function *f* that either changes its value in the actual state, or do not change their values in this state, but change their values in other states _[for example, discrete-time <<input,`inputs`>> in <<ContinuousTimeMode>>]_.
+* latexmath:[{\mathbf{v}_{\mathit{rest}}}] is the set of <<input>> variables and discrete-time states of function *f* which cannot be set by the importer in the current state, but in other states _[for example, discrete-time <<input,`inputs`>> in <<ContinuousTimeMode>>]_.
 
 _[The variable relationships are different in different states._
-_For example, during <<ContinuousTimeMode>>, a continuous-time output y does not depend on discrete-time <<input,`inputs`>> (because they are held constant between events)._
-_However, at <<EventMode>>, y depends on discrete-time <<input,`inputs`>>._
-_The function may compute the directional derivatives by numerical differentiation taking into account the sparseness of the equation system, or (preferred) by analytic derivatives.]_
+_For example, during <<ContinuousTimeMode>>, the partial derivate of a continuous-time output *y* with respect to discrete-time <<input,`inputs`>> is zero, because discrete-time <<input, `inputs`>> are constant between events._
```

Comment:
Wouldn't a discrte time input in ContinuouTimeModel belong to v_rest, and thus a partial derivative w.r.t. to this variable would not be defined (a call to get it not allowed) instead of being zero?